jest-rs/pom.xml

```diff
@@ -0,0 +1,141 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>jest-rs</artifactId>
+    <packaging>jar</packaging>
+    <name>Jest JAX-RS Client Jar</name>
+    <description>ElasticSearch Java REST client - JAX-RS Client package</description>
+    <url>https://github.com/searchbox-io/Jest</url>
+
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <scm>
+        <connection>scm:git:git@github.com:searchbox-io/Jest.git</connection>
+        <developerConnection>scm:git:git@github.com:searchbox-io/Jest.git</developerConnection>
+        <url>git@github.com:searchbox-io/Jest.git</url>
+    </scm>
+
+    <developers>
+        <developer>
+            <id>trajano</id>
+            <name>Archimedes Trajano</name>
+            <email>arch@trajano.net</email>
+        </developer>
+        <developer>
+            <id>TheKingOfTheEast</id>
+            <name>Dogukan Sonmez</name>
+            <email>dogukansonmez@yahoo.com</email>
+        </developer>
+        <developer>
+            <id>ferhatsb</id>
+            <name>Ferhat Sobay</name>
+            <email>ferhat@searchbox.io</email>
+        </developer>
+        <developer>
+            <id>kramer</id>
+            <name>Cihat Keser</name>
+            <email>info@cihatkeser.com</email>
+            <url>http://about.cihatkeser.com</url>
+        </developer>
+    </developers>
+
+    <parent>
+        <groupId>io.searchbox</groupId>
+        <artifactId>jest-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.apache.lucene</groupId>
+            <artifactId>lucene-test-framework</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.elasticsearch</groupId>
+            <artifactId>elasticsearch</artifactId>
+            <type>test-jar</type>
+        </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-all</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.carrotsearch.randomizedtesting</groupId>
+            <artifactId>randomizedtesting-runner</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.lucene</groupId>
+            <artifactId>lucene-codecs</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.searchbox</groupId>
+            <artifactId>jest-common</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.glassfish.main.extras</groupId>
```

Comment:
you're talking about glassfish?  That's to provide the implementation for the JAX-RS.  It is only for testing.  The JEE7 container will provide the actual implementation.